keyboards/converter/ps2_usb/matrix.c

```diff
@@ -0,0 +1,446 @@
+                }
+                break;
+            // following are states of Pause
+            case E1:
+                switch (code) {
+                    case 0x14:
+                        state = E1_14;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14:
+                switch (code) {
+                    case 0x77:
+                        state = E1_14_77;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14_77:
+                switch (code) {
+                    case 0xE1:
+                        state = E1_14_77_E1;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14_77_E1:
+                switch (code) {
+                    case 0xF0:
+                        state = E1_14_77_E1_F0;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14_77_E1_F0:
+                switch (code) {
+                    case 0x14:
+                        state = E1_14_77_E1_F0_14;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14_77_E1_F0_14:
+                switch (code) {
+                    case 0xF0:
+                        state = E1_14_77_E1_F0_14_F0;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            case E1_14_77_E1_F0_14_F0:
+                switch (code) {
+                    case 0x77:
+                        matrix_make(PAUSE);
+                        state = INIT;
+                        break;
+                    default:
+                        state = INIT;
+                }
+                break;
+            // Following are states of Control'd Pause
+            case E0_7E:
+                if (code == 0xE0)
+                    state = E0_7E_E0;
+                else
+                    state = INIT;
+                break;
+            case E0_7E_E0:
+                if (code == 0xF0)
+                    state = E0_7E_E0_F0;
+                else
+                    state = INIT;
+                break;
+            case E0_7E_E0_F0:
+                if (code == 0x7E)
+                    matrix_make(PAUSE);
+                state = INIT;
+                break;
+            default:
+                state = INIT;
+        }
+    }
+
+    // TODO: request RESEND when error occurs?
+/*
+    if (PS2_IS_FAILED(ps2_error)) {
+        uint8_t ret = ps2_host_send(PS2_RESEND);
+        xprintf("Resend: %02X\n", ret);
+    }
+*/
+    return 1;
```

Comment:
Please add `matrix_scan_quantum` here. 
```suggestion
    matrix_scan_quantum();
    return 1;
```